site/_docs/howto.md

```diff
@@ -684,8 +684,14 @@ Note: release artifacts (dist.apache.org and repository.apache.org) are managed
 
 Before you start:
 
+* Send an email to [dev@calcite.apache.org](mailto:dev@calcite.apache.org) notifying that RC build process
+  is starting and therefore `master` branch is in code freeze until further notice.
 * Set up signing keys as described above.
+* Upload your key to [https://keyserver.ubuntu.com](https://keyserver.ubuntu.com) and/or
+  [http://pool.sks-keyservers.net:11371](http://pool.sks-keyservers.net:11371) (keyservers used by Nexus).
```

Comment:
Move the 'upload your key' to the 'set up signing keys' section. Maybe add a troubleshooting point 'If xxx happens make sure that you uploaded your key to ubuntu'.

Reply:
Also notice that upload the key is only needed for the first time release manager.

Reply:
Changed.